sdk/core/Azure.Core.TestFramework/src/RecordedTestUtilities.cs

```diff
@@ -11,7 +12,7 @@ public static class RecordedTestUtilities
 
         public static RecordedTestMode GetModeFromEnvironment()
         {
-            string modeString = Environment.GetEnvironmentVariable(ModeEnvironmentVariableName);
```

Comment:
I don't see any change corresponding to this. It is explicitly passed as an environment variable from sdk/searc/tests.yml, and is explicitly to disable some/most track 1 tests that don't work with our infrastructure. As I see it, this would break live test runs.

Reply:
I'll move this to a separate PR.

Reply:
Note that this variable really doesn't have much to do with the actual test mode. Perhaps the name was probably chosen, but it really just means: this is a live test run, and some rest actually don't get compiled even (well, at least it was that way for Key Vault). It's safer to remove this change, or at least kick off a live test run to make sure all tests pass still.

Reply:
Not sure I understand. This is what recording framework uses to switch between recording and playback.

Reply:
ci.yml is hardcoded to pass a specific value that gets used in ways unrelated to the recording framework.

Reply:
Sure, I just wanted to add another way to pass this value for a local development. Don't see how that breaks the CI sceanrio.